taskvine/src/bindings/python3/ndcctools/taskvine/dask_executor.py

```diff
@@ -175,7 +175,7 @@ def get(self, dsk, keys, *,
             self.wrapper = wrapper
             self.wrapper_proc = wrapper_proc
             self.prune_files = prune_files
-            self.category_execution_time = defaultdict(list)
+            self.category_execution_time = defaultdict(lambda: {"num_tasks": 0, "avg_execution_time": 0})
```

Comment:
Remove the "avg_", as that would be confusing. You are keeping the total execution time, not the average.